dbhydra/dbhydra_core.py

```diff
@@ -496,22 +496,22 @@ def generate_table_dict(self):
         return (table_dict)
 
 
-class BigQueryDb:
-    def __init__(self, credentials_path, project_id):
+class BigQueryDb(AbstractDB):
+    def __init__(self, credentials_path, project_id, dataset_name):
         # super().__init__(None,None)
         self.credentials_path = credentials_path
         self.project_id = project_id
+        self.dataset = dataset_name
 
         self.credentials = service_account.Credentials.from_service_account_file(self.credentials_path, scopes=[
             "https://www.googleapis.com/auth/cloud-platform"], )
 
-        self.client = bigquery.Client(credentials=self.credentials, project=self.credentials.project_id, )
 
     def connect_remotely(self):
-        print("Connect remotely")
+        self.client = bigquery.Client(credentials=self.credentials, project=self.credentials.project_id)
 
     def connect_locally(self):
-        print("Connect locally")
+        self.client = bigquery.Client(credentials=self.credentials, project=self.credentials.project_id, )
```

Comment:
I suggest raising exception saying that there is no big query connector available instead of connecting to remote connection